include/erlcloud_aws.hrl

```diff
@@ -7,6 +7,8 @@
           s3_host="s3.amazonaws.com"::string(),
           s3_port=80::non_neg_integer(),
           s3_follow_redirect=false::boolean(),
+          s3_follow_redirect_count=2::non_neg_integer(),
+          s3_bucket_access_method=vhost::vhost|path,
```

Comment:
Amazon strongly suggest to create buckets with DNS-compliant names (see [bucket_restrictions] (http://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html)). BTW AWS UI doesn't allow users to create non-DNS-compliant buckets, but it can be done via command line ```aws s3api``` request. I think we should keep both method but probably it is worth changing default value of ```s3_bucket_access_method``` to ```path```. Thoughts?